husky_ws/docker/Dockerfile

```diff
@@ -1,5 +1,22 @@
+# Choose the architecture: arm or amd
+ARG TARGET_PLATFORM=amd
+
+# Base Image : https://hub.docker.com/r/arm64v8/ros/tags?page=1&name=humble
+FROM arm64v8/ros:humble AS arm
+
+# TODO: Verify whether the package needs to be installed or removed for the ARM platform.
+
 # Base Image : https://hub.docker.com/r/osrf/ros/tags?page=1&name=humble
-FROM osrf/ros:humble-desktop-full
+FROM osrf/ros:humble-desktop-full AS amd
+
+# Install Gazebo
+RUN apt-get update && apt-get install -y \
+    ros-$ROS_DISTRO-gazebo-ros-pkgs \
+    && rm -rf /var/lib/apt/lists/*
+
+##########################################################################################
+
+FROM $TARGET_PLATFORM
```

Comment:
If you want to build the image with `buildx`, you may refer to [here](https://github.com/YuZhong-Chen/ArduinoIDE/blob/7d16d17d45e56e9fa343564d115f7bfb19d8e6e9/.github/workflows/action.yaml#L34-L54).

I believe that using `buildx` is preferable to `docker manifest`, as the `docker manifest` command is experimental and may change between Docker releases without warning, or it could potentially be removed entirely in a future release. [Ref](https://docs.docker.com/reference/cli/docker/manifest/)

Reply:
Thanks! I believe the commands fit my needs. I'll test them out when configuring the automated build process.